Source/_AIInterface/ActionMoverMsg.cpp

```diff
@@ -1128,7 +1110,7 @@ void	Test( RECT r, Point p, Point.d )
 // return: attacker�� ������ 
 CMover* CActionMover::PreProcessDamage( CMover* pMover, OBJID idAttacker, BOOL bTarget, int nReflect )
 {
-	if( pMover->m_dwMode & MATCHLESS_MODE )		// �����̸� ���⼭ ����.	
+	if ((pMover->m_dwMode & MATCHLESS_MODE) && pMover->IsAuthHigher(AUTH_GAMEMASTER2))		// �����̸� ���⼭ ����.	
```

Comment:
```suggestion
	if (pMover->m_dwMode & MATCHLESS_MODE)		// �����̸� ���⼭ ����.	
```

I understand why in the context of a live server. But in a context of a base source, and servers developed from it, MATCHLESS_MODE may be legit for people < GAMEMASTER2